src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -152,11 +154,101 @@ public void setPerson(Person target, Person editedPerson) {
         public ObservableList<Person> getFilteredPersonList() {
             throw new AssertionError("This method should not be called.");
         }
+        @Override
+        public ObservableList<Leave> getFilteredLeaveList() {
+            throw new AssertionError("This method should not be called.");
+        }
 
         @Override
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        /**
+         * Replaces leave book data with the data in {@code leavesBook}.
+         *
+         * @param leavesBook
+         */
+        @Override
+        public void setLeavesBook(ReadOnlyLeavesBook leavesBook) {
+
```

Comment:
Will an implementation be provided for these methods with empty bodies or are they supposed to throw an assertion error?

Reply:
Added the implementation, sorry i forgot to add.